akka-testkit-typed/src/test/scala/akka/testkit/typed/scaladsl/BehaviorTestKitSpec.scala

```diff
@@ -114,6 +114,16 @@ class BehaviorTestKitSpec extends WordSpec with Matchers {
       spawnAnonymous.props should ===(Props.empty)
     }
 
+    "allow expecting NoEffects by type" in {
+      val testkit = BehaviorTestKit[Father.Command](Father.init)
+      testkit.expectEffectType[NoEffects.type]
```

Comment:
That's somewhat weird, and I guess we want people to use the other syntax?

Reply:
could we use something like this:
```
  trait NoEffects extends Effect
  
  case object NoEffects extends NoEffects
```

Reply:
That's much better, thank you. I did it.